packages/vite/src/node/plugins/json.ts

```diff
@@ -87,7 +87,7 @@ export function jsonPlugin(
             }
 
             return {
-              code: `export default JSON.parse(${JSON.stringify(json)})`,
+              code: `export default /* #__PURE__ */ JSON.parse(${JSON.stringify(json)})`,
```

Comment:
Good catch! I think the one below also needs pure comment. 

https://github.com/vitejs/vite/blob/e690d8bb1e5741e81df5b7a6a5c8c3c1c971fa41/packages/vite/src/node/plugins/json.ts#L123

This only kicks in when there is a large object in a top level key value, something like:

```json
{
  "large": { "key": "....10000 length string..." }
}
```